react/src/components/organisms/TabContainer/tab.js

```diff
@@ -31,15 +35,53 @@ class Tab extends React.Component {
             this.defaultSet = true;
           }
           return(
-            <button
-              className={tabClasses}
-              onClick={(e) => {
-                this.scrollIntoView(e.target);
-                setActiveTab(this.tabIdent, this.props.children)
-              }}
-            >
-            {this.props.title}
-            </button>
+            <li role="presentation" className={tabClasses}>
+              <button
+                role="tab"
+                id={this.tabIdent}
+                onClick={(e) => {
+                  this.scrollIntoView(e.target);
+                  setActiveTab(this.tabIdent, this.props.children)
+                }}
+                aria-selected={active}
+                tabIndex={!active ? -1 : 0}
+                aria-describedby={this.props.a11yID}
+
+
+    // THIS EVENT IS ONLY AVAILABLE TO THE ONFOCUS TAB.
+                onKeyDown = {(e) => {
```

Comment:
For functions of this size, it's better to define an arrow function which you could then set this to. Something like:
`onKeyDown = {this.handleKeyDown}`